assessment_validation.tf

```
@@ -9,6 +9,18 @@
 # that contains a precondition.  The precondition is evaluated at plan time, and
 # if it fails, the plan will fail with the error message provided.
 
+# NOTE: When adding new validation checks, be sure to include them to the
+# initial apply in the terraform_apply.sh script.
+
+resource "null_resource" "validate_assessment_account_name_matches_workspace" {
+  lifecycle {
+    precondition {
+      condition     = replace(replace(lower(var.assessment_account_name), "/[()]/", ""), " ", "-") == terraform.workspace
+      error_message = "Assessment account name (${var.assessment_account_name}) does not align with the currently-selected workspace (${terraform.workspace}).  Are you sure that you are using the correct tfvars file?"
```

I think this language is a little clearer, but you can ignore this if you want:
```suggestion
      error_message = "Assessment account name (${var.assessment_account_name}) does not agree with the currently-selected workspace (${terraform.workspace}).  Are you sure that you are using the correct tfvars file?"
```